pkg/database/mysql.go

```diff
@@ -0,0 +1,26 @@
+package database
+
+
+type MySQLDatabase struct {
+	DB *bun.DB
+}
+
+var (
+	once sync.Once
+	dbInstance *MySQLDatabase
+)
+
+func NewMySQLDatabase(conf *entities.Config) *MySQLDatabase {
+	once.Do(func() {
+		sqlDB, err := sql.Open("mysql", fmt.Sprintf("%s:%s@tcp(%s:%s)/%s", conf.DBUser, conf.DBPassword, conf.DBHost, conf.DBPort, conf.DBName))
+		if err != nil {
+			panic(err)
+		}
+		dbInstance = &MySQLDatabase{DB: bun.NewDB(sqlDB, mysqldialect.New())}
+	})
+	return dbInstance
+}
+
+func (MySQLDatabase *MySQLDatabase) GetDB() *bun.DB {
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider shorter receiver parameter name</b></div>
<div id="fix">

Consider renaming the receiver parameter from `MySQLDatabase` to a shorter name like `db` or `m`. The current name is redundant with the type name and makes the code less readable.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<blockquote>Check the AI-generated fix before applying</blockquote>
<div id="code">


```suggestion
func (db *MySQLDatabase) GetDB() *bun.DB {
```

</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/aakashgusain1990/productivite/pull/14#issuecomment-2563464674>#148ccb</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged